core/crypto/encrypt.go

```diff
@@ -104,7 +138,7 @@ func EncryptToWriterAes(out io.Writer, data string, rootKeyTriplet []string) err
 
 	lastEncryptToWriterAesCall = time.Now()
 
-	_, _, aesKey := rootKeyTriplet[0], rootKeyTriplet[1], rootKeyTriplet[2]
+	aesKey := rkt.AesSeed
```

Comment:
This is much better than trying to memorize which array index belongs to which attribute.